tests/images.spec.ts

```diff
@@ -18,3 +28,60 @@ test("search for custom image and create an instance from it", async ({
 
   await deleteInstance(page, customInstance);
 });
+
+test("Export an image", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  const imageAlias = await createImage(page, instance);
+
+  await page.goto("/ui/");
+  await page.getByRole("link", { name: "Images", exact: true }).click();
+  await page.getByPlaceholder("Search").click();
+  await page.getByPlaceholder("Search").fill(imageAlias);
+  page.getByText(imageAlias);
+  const imageName = await getImageNameFromAlias(page, imageAlias);
+  await page.getByLabel("export image").click();
+  await page.waitForSelector(
+    `text=Image ${imageName} was successfully downloaded. Please check your downloads folder.`,
+  );
+
+  await deleteImage(page, imageName);
+  await deleteInstance(page, instance);
+});
+
+test("Upload an image", async ({ page }) => {
+  const IMAGE_FILE = "./tests/fixtures/e2e-test-image.tar.gz";
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  await visitInstance(page, instance);
+  const instanceBaseImageName = await page
+    .locator("text=Base image")
+    .locator("xpath=following-sibling::*")
+    .innerText();
+
+  await page.goto("/ui/");
+  await page.getByRole("link", { name: "Images", exact: true }).click();
+
+  await page.getByRole("button", { name: "Upload image" }).click();
+  await page.getByLabel("Image backup file").setInputFiles(IMAGE_FILE);
+  await page.getByPlaceholder("Enter alias").fill(`${instance}-1`);
+  await page
+    .getByLabel("Import image from file")
+    .getByRole("button", { name: "Upload image" })
+    .click();
+  await page.waitForSelector(`text=Image uploaded.`);
+
+  const uploadedImageName = await getImageNameFromAlias(page, `${instance}-1`);
+  await deleteImage(page, uploadedImageName);
+  await deleteImage(page, instanceBaseImageName);
+  await deleteInstance(page, instance);
+});
+
+test("Delete an image", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+  const imageAlias = await createImage(page, instance);
+  const imageName = await getImageNameFromAlias(page, imageAlias);
+  await deleteImage(page, imageName);
+  await deleteInstance(page, instance);
+});
```

Comment:
```suggestion
```
I think deletion is implicitly covered by the "Upload an image" case above.

Reply:
Alrighty, I'll delete this function :')